data/repos.yml

```diff
@@ -590,21 +764,61 @@
   team: "#govuk-platform-reliability-team"
   type: Utilities
 
+# Same support model as govuk-user-reviewer, i.e. _really_ just maintained by 2nd line / leads
+# I'll probably add this in a separate PR.
+- repo_name: govuk-rota-generator
+  team: "#govuk-platform-reliability-team"
+  type: Utilities
+  sentry_url: false
+  dashboard_url: false
+
 - repo_name: govuk-rfcs
   team: "#govuk-developers"
   type: Utilities
 
+# Check with Roch to see if this is still used
+# (fork with 1 extra commit: https://github.com/alphagov/govuk-ruby/commit/07a14ea112ada861c95efb1ac55881c51f0e830e)
+# I can't see a corresponding commit in govuk-puppet
+# This will go away with Replatforming though, so is ripe for archiving soon anyway.
+- repo_name: govuk-ruby
+
+# Actively used, should be added to this list
+- repo_name: govuk-ruby-images
+  type: Utilities
+  team: "#govuk-replatforming"
+  sentry_url: false
+  dashboard_url: false
+
 - repo_name: govuk-rota-announcer
   private_repo: true
   team: "#govuk-platform-reliability-team"
   type: Utilities
 
+# Looks actively used
+- repo_name: govuk-s3-mirror
+  private_repo: true # internal
+  team: "#data-products"
+  type: Data science
+  sentry_url: false
+  dashboard_url: false
+
 - repo_name: govuk-saas-config
   team: "#govuk-platform-reliability-team"
   type: Utilities
   sentry_url: false
   dashboard_url: false
 
+# Propose that User Experience and Measurement adds this in a separate PR
+- repo_name: govuk-search-relevance-tool
```

Comment:
@gclssvglx

Reply:
Ah that's me. It's an old prototype that I recreated for GIFT week at Tara's request. I will archive it again cc @sihugh

Reply:
Thanks, please let me know when that's done @hannako 👍

Reply:
Just nudging on this @hannako